lib/spawn.ml

```diff
@@ -12,8 +12,9 @@ let proto_version = 0
 let prefer_sock = Sys.os_type = "Win32"
 let accept_timeout = 10.0
 
-let pr_err s = Printf.eprintf "(Spawn  ,%d) %s\n%!" (Unix.getpid ()) s
-let prerr_endline s = if CDebug.is_debug_all() then begin pr_err s end else ()
+let debug_spawn = CDebug.create ~name:"spawn"
```

Comment:
Here there is a regression, the PID is very important to know.
I did not check if there is a regression also there, but also the STM debug print should contain the PID, and this is important to undestand who is speaking when there are multple processes